util.h

```diff
@@ -13,28 +13,28 @@ typedef struct {
 
 #define INIT_SIZE 8
 
-static inline String *make_string(void)
+static inline String make_string(void)
 {
-    String *r = malloc(sizeof(String));
-    r->body = malloc(INIT_SIZE);
-    r->nalloc = INIT_SIZE;
-    r->len = 0;
-    r->body[0] = '\0';
-    return r;
+    char *body = malloc(INIT_SIZE);
+    body[0] = '\0';
```

Comment:
Replace `malloc` with `calloc` to ensure allocated memory filled with bytes of value zero.